SpreadCheetah.SourceGenerator.Test/Tests/ColumnHeaderTests.cs

```diff
@@ -0,0 +1,126 @@
+using System.Reflection;
+using SpreadCheetah.SourceGeneration;
+using SpreadCheetah.SourceGenerator.Test.Models.ColumnHeader;
+using Xunit;
+
+namespace SpreadCheetah.SourceGenerator.Test.Tests;
+
+public class ColumnHeaderTests
+{
+    [Fact]
+    public void ColumnHeader_ClassWithPropertyReferenceColumnHeaders_CanReadTypeAndPropertyName()
```

Comment:
I really appreciate that you wrote tests as well 👍  However, I don't think we need all these tests, especially since the code we are testing is very simple (basically just property assignments in constructors). I think it would be enough to have a single test for each attribute constructor, and one assertion per property.

Reply:
Done.